CHANGELOG.md

```diff
@@ -6,17 +6,11 @@
 
 - Migrate `SelectCountryScreen` to a fragment
 - Inject `DrugFrequencyToLabelMap` directly in `TeleconsultMedicinesView` instead of handling it in the effect handler
```

Comment:
Done.

Reply:
Can we add the inject changelog in the demo changes as well?

> - Inject `DrugFrequencyToLabelMap` directly in `TeleconsultMedicinesView` instead of handling it in the effect handler